docs/source/conf.py

```diff
@@ -5,7 +5,10 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
 import datetime
 import emoji
-
+from pathlib import Path
```

Comment:
Instead of trying to hack `conf.py` you can change the `build` command that RTD uses in `.readthedocs.yaml` (https://docs.readthedocs.io/en/stable/config-file/v2.html#build-commands)

Reply:
Actually, there's an example of exactly the thing we want to do:

```python
python:
  install:
    - requirements: docs/requirements.txt
    # Install our python package before building the docs
    - method: pip
      path: .
```

Reply:
N.B. I did previously try using

```
build:
  command: hatch build
```

but this skips some of the extras that RTD do for you (e.g. building PDF versions of the docs).

Reply:
> Actually, there's an example of how to do exactly the thing we want to do in the `python` section:
> 
> ```python
> python:
>   install:
>     - requirements: docs/requirements.txt
>     # Install our python package before building the docs
>     - method: pip
>       path: .
> ```

That did the job, working now!